chalicelib/checks/qc_checks.py

```diff
@@ -180,6 +180,64 @@ def identify_files_without_qc_summary_bed(connection, **kwargs):
     return check
 
 
+@check_function(file_type=None, status=None, search_add_on=None)
+def identify_files_without_qc_summary_rnaseq(connection, **kwargs):
+    t0 = time.time()  # keep track of how start time
+    time_limit = 270  # 4.5 minutes
+    check = CheckResult(connection, 'identify_files_without_qc_summary_rnaseq')
+    # must set this to be the function name of the action
+    check.action = 'patch_quality_metric_summary_rnaseq'
+    default_filetype = 'FileProcessed'  # skip fastq
+    default_stati = 'released%20to%20project&status=released&status=uploaded&status=pre-release'
+    filetype = kwargs.get('file_type') or default_filetype
+    stati = 'status=' + (kwargs.get('status') or default_stati)
+    search_query = 'search/?type={}&{}&frame=object'.format(filetype, stati)
+    search_query += '&track_and_facet_info.experiment_type=RNA-seq'
+    search_query += '&file_type=gene+expression&file_type=read+positions'
```

Comment:
You're free to use `ff_utils.search_metadata` but I would definitely recommend looking at `ff_utils.faceted_search` as it can make building these queries a lot cleaner.

Reply:
Good to know. Thanks! I will merge this one for now since it's working (no more cases to test for now), but for the future I will definitely try faceted_search.